### ISSUE TITLE
Metric computation in training phase

### ISSUE THREAD

**Issue by guoyang9** [Cyanogenoid/pytorch-vqa#16]
Hi,
I read your code from start to finish and found it very clear and modular. 
Based on your implementation, with some minor modifications, an overall accuracy result of 60.16 (original paper is 59.67) on VQA v2.0 validation set as well as 64.99 (original paper is 64.5) on VQA v1.0 test-dev were observed! It's very impressive and promising which means we can easily design our own algorithm on this code.
But there is a small issue confusing me (maybe stupid). That the computation of two metrics (loss and acc) in the training phase doesn't make much sense. What does the momentum do? How about just using the batch loss and acc?
Thanks.

**Comment by Cyanogenoid** [comment]
The batch loss and acc aren't that useful to look at directly because it can vary a lot between individual batches, so it is hard to figure out by how much it's improving. The momentum just smooths it out a bit to make it easier to view.

**Comment by guoyang9** [comment]
I see. The pytorch tensorboard with self-smoothing can be more visually intuitive. 
Thanks for the code.
